sourcegraph/about-replatform
# Add dev onboarding guide

Closes #79 by adding the guides pages directory, the onboarding guide, and related assets.

## To Test
- Check that the page looks and functions correctly on desktop/mobile
- Check static builds
